state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -380,22 +362,17 @@ pub mod pallet {
 
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
-		pub minimum_swap_amounts: Vec<(Asset, AssetAmount)>,
 		pub _phantom: PhantomData<T>,
 	}
 
 	#[pallet::genesis_build]
 	impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
-		fn build(&self) {
-			for (asset, min) in &self.minimum_swap_amounts {
-				MinimumSwapAmount::<T>::insert(asset, min);
-			}
-		}
+		fn build(&self) {}
```

Comment:
I'm pretty sure we can just delete the GenesisConfig struct and associated impls entirely.